src/main/java/com/bookstore/service/user/UserServiceImpl.java

```diff
@@ -32,4 +41,14 @@ public UserResponseDto register(UserRegistrationRequestDto userRegistrationReque
         user.setRoles(Set.of(roleRepository.findByRoleName(DEFAULT_ROLE)));
         return userMapper.toDto(userRepository.save(user));
     }
+
+    @Override
+    public UserLoginResponseDto login(UserLoginRequestDto userLoginRequestDto) {
```

Comment:
authentication logic should be in different service
for example AuthenticationService

Reply:
Should the registration logic also be in another service? I think the HW says that the registration should be created on the UserService but I think it is better to have user-related logic (e.g. getUsers) on the UserService and everything else such as the registration and the login logic on the AuthenticationService